src/otx/core/model/detection_3d.py

```diff
@@ -397,7 +423,7 @@ def _customize_inputs(
         self,
         entity: Det3DBatchDataEntity,
     ) -> dict[str, Any]:
-        img_sizes = np.array([img_info.ori_shape for img_info in entity.imgs_info])
+        img_sizes = np.array([img_info.ori_shape[::-1] for img_info in entity.imgs_info])
```

Comment:
Is there any reason why setting img_size to WxH?

Reply:
I will change it to use HxW

Reply:
forward function and outputs post processing expect WxH image ratio. I will try to change this though

Reply:
Aligned to HxW everywhere https://github.com/openvinotoolkit/training_extensions/pull/4046/commits/ae60b71090bdc8f7e00776055de3f2c7e9684831